components/src/molecules/collapse/Collapse.html

```diff
@@ -1,20 +1,27 @@
 <div class="a-collapse">
-  <button
-    :class="['a-collapse__button', titleClass]"
-    :aria-controls="collapseLocal"
-    @click="toggle"
+  <!-- @slot Custom button slots (Scoped slot) -->
+  <slot
+    name="button"
+    :toggle="toggle"
+    :collapseLocal="collapseLocal"
+    :ariaControls="ariaControls"
+  >
+  <a-button
+    class="a-collapse__button a-button--blank"
+    :aria-controls="ariaControls"
+    @click.native="toggle"
     :aria-expanded="!collapseLocal"
   >
-    <!-- @slot Custom title slots (Named slot) -->
-    <slot name="title">
       {{ title }}
       <a-icon
+        title="Angle down"
```

Comment:
Btw this title should be in prop for supporting translation, but I think we can skip if for now. Afaik we'll change soon `Icon` component and will be best to have the icon title defined in `Icon` component level.